src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -18,6 +18,13 @@ class LaravelLogViewer
      */
     private $folder;
 
+    private static $patterns = [
+        'all_log_data' => '/\[\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}([\+-]\d{4})?\].*/',
+        'current_log_data_first' => '/^\[(\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}([\+-]\d{4})?)\](?:.*?(\w+)\.|.*?)',
+        'current_log_data_second' => ': (.*?)( in .*?:[0-9]+)?$/i',
+        'get_files' => '/\{.*?\,.*?\}/i',
```

Comment:
About keys consistency: 

You suggest: `all_log_data`, `current_log_data_first`, `current_log_data_second` and `get_files`. There is a mix of verbs and descriptive words (and plural vs singular)

What do you think of `logs`, `current_log` and `files`? (there is still a mix of singular and plural though...)

Reply:
I like your suggestion :)

Reply:
OK so go on if you want!